x/oracle/client/cli/txRegisterOracleVote.go

```diff
@@ -0,0 +1,55 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func CmdRegisterOracleVote() *cobra.Command {
```

Comment:
I have a question about the CLI. In my opinion, the vote should be done automatically, not the user's cli. Why is the additional cli provided?
Except for the question, it's lgtm!

Reply:
Yes, this will usually happen automatically.
This is a CLI created for the purpose of development.
If we don't have this CLI, we have to run panacea-doracle every time to test it.

Reply:
Ah, I got it. Thanks!